OrderService/src/main/resources/application.properties

```
@@ -0,0 +1,4 @@
+spring.application.name=OrderService
```

did this to get it running
```suggestion
spring.application.name=OrderService
server.port=${ORDER_SERVER_PORT}
container.port=${ORDER_CONTAINER_PORT}
mysql.server.port=${TRANSACTION_MYSQL_SERVER_PORT}
#IF CHANGING CHANGE WITH spring.datasource.url in the main file for this
mysql.container.port=${TRANSACTION_MYSQL_CONTAINER_PORT}
#IF CHANGING CHANGE WITH mysql.container.port in the ports file for this
spring.datasource.url=jdbc:mysql://orderdb:3306/order_service
spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
#spring.cloud.config.name=transaction
#spring.profiles.active=${ENV}
#spring.config.import=configserver:http://${USER_NAME}:${USER_PSWD}@config-service:8888
```